src/cms-content/learn/index.ts

```diff
@@ -4,7 +4,6 @@ import glossary from './learn-glossary.json';
 import landing from './learn-landing.json';
 import caseStudies from './learn-case-studies.json';
 import productsLanding from './products-landing.json';
-import productPages from './products-full-pages.json';
```

Comment:
What’s up with products-full-pages.json?  Looks like we removed some references to it.  Should the file be deleted?

Reply:
yes!